crates/core/tedge_watchdog/src/systemd_watchdog.rs

```diff
@@ -101,8 +101,6 @@ async fn start_watchdog_for_tedge_services(tedge_config_dir: PathBuf) {
         "tedge-mapper-aws",
         "tedge-mapper-collectd",
         "tedge-agent",
-        "tedge-log-plugin",
-        "tedge-configuration-plugin",
```

Comment:
I think some users are using, or at least are starting to. Though there are some other issues/limitations on how this is being done at the moment.

* https://github.com/thin-edge/thin-edge.io/issues/2355

So we can look at the watchdog improvements in another issue. Otherwise we could consider removing it and direct users to use some other service level supervision, such as [monit](https://mmonit.com/monit/)